libecl/include/ert/ecl/ecl_nnc_export.h

```diff
@@ -38,6 +38,8 @@ typedef struct {
   int grid_nr2;
   int global_index2;
 
+  int nnc_index;
```

Comment:
Perhaps, I chose the name nnc_index, as that was the name used throughout the grid processing code, as well as in the name of the list (nnc_index_list). Your choice: if you want me to change it I will.